lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs

```diff
@@ -1087,5 +1220,24 @@ private string GetEndPointFromContext(Optional<IActiveContext> context)
             } 
             return context.Value.EvaluatorDescriptor.NodeDescriptor.HostName; 
         }
+
+        /// <summary>
+        /// Ensure the Timer is disposed when the driver object is deleted
+        /// </summary>
+        ~IMRUDriver()
+        {
+            if (_timeoutMonitorTimer != null)
```

Comment:
given it is not called  frequently, yes, we can remove the double null checking.